packages/fluent-editor/src/modules/ai/constants.ts

```
@@ -0,0 +1,44 @@
+export const SELECT_PLACEHOLDER = '向我提问/选择操作';
+export const STOP_ANSWER = '停止回答';
+export const REPLACE_SELECT = '替换选中内容';
+export const INSERT_TEXT = '插入内容';
+export const INSERT_SUB_CONTENT_TEXT = '插入内容下方';
+export const REGENERATE = '重新生成';
+export const CLOSE = '关闭';
+export const THINK_TEXT = '正在为您分析并总结答案';
+export const RESULT_HEADER_TEXT = '根据您的诉求，已为您解答，具体如下：';
+
+export const MENU_TITLE_DATA = {
+    editor: [
```

ESLint

[error] Expected indentation of 2 spaces but found 4.

(style/indent)